src/controllers/ArtsController.js

```diff
@@ -271,6 +275,157 @@ class ArtsController {
       return res.status(response.code).json(response);
     }
   }
+
+  /**
+   * @desc GET /api/v1/articles
+   * @param {object} req
+   * @param {object} res
+   * @memberof ArtsController
+   * @returns {Object} All Articles
+   */
+  static async getAllArticles(req, res) {
+    try {
+      const { limit, page } = req.query;
+      const limitDefault = limit || 20;
+      const pageDefault = page || 1;
+
+      const offset = limitDefault * (pageDefault - 1);
+
+      const allArticlesCount = await Art.findAndCountAll();
+
+      const pages = Math.ceil(allArticlesCount.count / limitDefault);
+
+      const articles = await Art.findAll({
+        include: [
+          {
+            model: Category,
+            as: 'Category',
+            attributes: ['id', 'categoryName'],
+          },
+          {
+            model: User,
+            as: 'Author',
+            attributes: ['username'],
+          },
+          {
+            model: Comment,
+            attributes: [],
+          }
+        ],
+        order: [
+          ['createdAt', 'DESC'],
+          ['id', 'DESC'],
+        ],
+        attributes: [
+          'id',
+          'slug',
+          'artistId',
+          'categoryId',
+          'title',
+          'description',
+          'featuredImg',
+          'createdAt',
+          [sequelize.fn('COUNT', 'comments.id'), 'commentsCount'],
+        ],
+        group: ['Art.id'],
+        limit: limitDefault,
+        offset,
+      });
+
+      response = new Response(
+        'Ok',
+        200,
+        'All Articles',
+        {
+          articles,
+          articlesGrandTotal: allArticlesCount.count,
+          page: pageDefault,
+          pages
+        }
+      );
+      return res.status(response.code).json(response);
+    } catch (err) {
+      response = new Response(
+        'Not ok',
+        500,
+        `${err}`,
+      );
+      return res.status(response.code).json(response);
+    }
+  }
+
+  /**
+   * @desc GET /api/v1/articles/slug
+   * @param {object} req
+   * @param {object} res
+   * @memberof ArtsController
+   * @returns {Object} Single Article
+   */
+  static async getSingleArticle(req, res) {
+    try {
+      const { slug } = req.params;
+
+      const article = await Art.findOne({
+        where: { slug },
+        include: [
+          {
+            model: Category,
+            as: 'Category',
+            attributes: ['id', 'categoryName'],
+          },
+          {
+            model: Comment,
+            attributes: ['id', 'userId', 'body', 'createdAt'],
+          },
+          {
+            model: User,
+            as: 'Author',
+            attributes: ['username'],
+          },
+          {
+            model: Media,
+            as: 'Media',
+            attributes: ['id', 'contentUrl'],
+          },
+        ],
+        attributes: [
```

Comment:
Instead of  listing all the attributes like this, you can exclude the ones you dont want. 

```
attributes: { exclude: ['updatedAt', ...etc] }

Reply:
Okay, will do